src/cryptonote_core/blockchain.cpp

```diff
@@ -4448,6 +4544,7 @@ bool Blockchain::check_blockchain_pruning()
   return m_db->check_pruning();
 }
 //------------------------------------------------------------------
+// returns min(Mb, 2Ml) as per https://github.com/ArticMine/Monero-Documents/blob/master/MoneroScaling2021-02.pdf from HF_VERSION_LONG_TERM_BLOCK_WEIGHT
```

Comment:
AFAICT it's returning min(Mb, 1.7 x) with x being max(long term median, 300000). and seems to be assuming that median(max(x,y)) == max(median(x),y), which I think is true (a test program I've just made seems to corroborate). So I'm not sure why you think this functoin should return something else. Do you disagree with how it's used maybe ?

Reply:
No I think it's ok given how it's used and that this is ok. Was just a little confused by it since it seemed a little different and wanted to double check